k8s.io/configmap-nginx.yaml

```diff
@@ -249,11 +249,11 @@ data:
 
         # Please update README.md when you update the list below.
         location / {
-          rewrite ^/bounty$          https://github.com/kubernetes/kubernetes.github.io/issues?q=is%3Aopen+is%3Aissue+label%3ABounty redirect;
```

Comment:
The redirect yields no results. 

Moreover, https://github.com/search?q=org%3Akubernetes+org%3Akubernetes-sigs+org%3Akubernetes-incubator+org%3Akubernetes-csi+org%3Akubernetes-client+is%3Aissue+label%3A%22Bounty%22&type=Issues is also empty.

Also, we are still hashing out our bug bounty program, so I think it will be useful to add it up later. (https://github.com/kubernetes/community/pull/2620)